src/classes/Policy.js

```diff
@@ -100,35 +100,42 @@ class Policy {
 	 * @param  {number} tab_id 		tab id
 	 * @param  {string} tab_host 	tab url host
 	 * @param  {string} tab_url 	tab url
-	 * @return {boolean}
+	 * @return {Object} 			{block, ss_unblock}
 	 */
 	shouldBlock(app_id, cat_id, tab_id, tab_host, tab_url) {
 		if (globals.SESSION.paused_blocking) {
-			return false;
+			return { block: false };
```

Comment:
we should return `{block, ss_unblock`} in all cases to prevent `undefined` errors

Reply:
You mean if we ever call shouldBlock elsewhere (but we don't)

Reply:
I'll make the requested changes though.